Library/Formula/slib.rb

```diff
@@ -0,0 +1,44 @@
+class SchemeInstalled < Requirement
+  fatal true
+
```

Comment:
I personally prefer Gauche. Is it okay to put a line `default_formula "gauche"` in this formula?